GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Spaceplanes.cfg

```diff
@@ -1696,7 +1696,7 @@
 		name = ModuleFuelTanks
 		volume = 100
 		basemass = -1
-		type = ServiceModule
+		type = Fuselage
```

Comment:
This is what I'm talking about. Why is this here, it changes the same thing the other pr does which is resource related for tacls. That pr is not going to be accepted anyway based on what pap said.